### DISCUSSION TITLE
Is there a proxy environment variable like CAPTCHA_SOLVER for instance???

### DISCUSSION THREAD

**Discussion by jscoys** [FlareSolverr/FlareSolverr#323] [Q&A]
Can't find anything on that topic, is there a way to specific the proxy url when launching a docker container, like CAPTCHA_SOLVER for instance? Can't find anything, please help!

**Comment by ngosang** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/300#issuecomment-1100736796
